thruster_manager:
  ros__parameters:
    max_force: 60.0
    max_torque: 40.0
    motor0:
      heave: 0.0
      pitch: -0.0
      roll: -0.0
      surge: -0.284934
      sway: -0.284934
      yaw: 0.58863
    motor1:
      heave: 0.0
      pitch: 0.0
      roll: -0.0
      surge: -0.482573
      sway: 0.224534
      yaw: -1.106754
    motor2:
      heave: 0.25
      pitch: -0.973376
      roll: -0.677881
      surge: 0.0
      sway: -0.0
      yaw: 0.0
    motor3:
      heave: -0.25
      pitch: 0.973376
      roll: -0.677881
      surge: -0.0
      sway: -0.00
      yaw: -0.0
    motor4:
      heave: -0.25
      pitch: -0.973376
      roll: 0.677881
      surge: 0.00
      sway: 0.00
      yaw: 0.0
    motor5:
      heave: 0.0
      pitch: 0.0
      roll: -0.0
      surge: 0.224534
      sway: -0.482573
      yaw: -1.106754
    motor6:
      heave: -0.0
      pitch: 0.0
      roll: 0.0
      surge: 0.422173
      sway: 0.422173
      yaw: 0.58863
    motor7:
      heave: 0.25
      pitch: 0.973376
      roll: 0.677881
      surge: -0.0
      sway: 0.00
      yaw: -0.0
    num_motors: 8
